### ISSUE TITLE
fail to get go version using ltc/scripts/install

### ISSUE THREAD

**Issue by guanglinlv** [cloudfoundry-incubator/lattice#131]
hi

I have follow <a href="https://github.com/cloudfoundry-incubator/lattice/tree/master/ltc#installing-from-source">the guide</a> to install ltc from source.however,the install script cant get go version with `grep -o "\d.\d.\d"` at <a href="https://github.com/cloudfoundry-incubator/lattice/blob/master/ltc/scripts/install#L5">#L5</a>.

my environment:

```shell
root@lattice-cli:~/workspace/lattice# cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=12.04
DISTRIB_CODENAME=precise
DISTRIB_DESCRIPTION="Ubuntu 12.04.4 LTS"
```

then i use `grep -o "[0-9]\+\.[0-9]\+\.[0-9]\+"` to instead of `grep -o "\d.\d.\d"`, it work...

**Comment by menicosia** [comment]
@guanglinlv the team switched `"\d.\d.\d"` to `"[[:digit:]]\.[[:digit:]]\.[[:digit:]]"` which doesn't look precisely like what you were looking for. (In particular, I'm not sure how this will work when we get to v0.2.10) do you want to provide feedback before I move forward with this change?

--
  Marco Nicosia
  Product Manager
  Pivotal Software, Inc.

**Comment by guanglinlv** [comment]
hi @menicosia ,I just puzzled about `"\d.\d.\d"` here,because it can't work in my ubuntu. this change is precisely good for me. 

thanks.

**Comment by davidwadden** [comment]
This was included with [v0.2.5](https://github.com/cloudfoundry-incubator/lattice/releases/tag/v0.2.5).  Closing this.

Thanks for reporting this issue!
David
